server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/HcfsType.java

```diff
@@ -45,6 +47,8 @@ public String normalizeDataSource(String dataSource) {
     // as it uses SSL for communication to Azure servers
     WASBS;
 
+    protected static final Logger LOG = LoggerFactory.getLogger(HcfsType.class.getName());
```

Comment:
To make is subclass specific:
```suggestion
    protected Logger LOG = LoggerFactory.getLogger(this.getClass());
```